tamoc/ambient.py

```diff
@@ -87,6 +87,9 @@
 
 import matplotlib.pyplot as plt
 
+import warnings
```

Comment:
This is so I can suppress some MPL warnings -- not sure why we're getting thoise, but they are annoying in a Notebook.